x-pack/legacy/plugins/lens/public/drag_drop/providers.test.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useContext } from 'react';
+import { mount } from 'enzyme';
+import { RootDragDropProvider, DragContext } from './providers';
+
+jest.useFakeTimers();
+
+describe('RootDragDropProvider', () => {
+  test('provides a consistent context across render cycles', () => {
```

Comment:
The contents of this test are fine, but maybe the description here could be updated to something like "reuses contexts for each render"?